doc/lspconfig.txt

```diff
@@ -382,9 +382,9 @@ Launches the requested (configured) client, but only if it successfully
 resolves a root directory. Note: Defaults to all configured servers matching
 the current buffer filetype.
 
-:LspStop [client_id]                                                *:LspStop*
-Stops the server with the given client id. Defaults to stopping all servers
-active on the current buffer. To force stop a language server: >
+:LspStop [client_id] or [client_name]                              *:LspStop*
+Stops the server with the given client id or client name. Defaults to stopping
```

Comment:
"client name" is not mentioned anywhere else in the help. I think we should generally refer to "config names", not "server names" or "client names" except for specific cases where we actually mean the client or server. These names do not refer to the literal client or server (which could have a different name than the config name).
```suggestion
:LspStop [client_id] or [config_name]                              *:LspStop*
Stops the server with the given client-id or config name. Defaults to stopping
```